hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -82,6 +85,7 @@ public MerkleAccountState(
 		this.hbarBalance = hbarBalance;
 		this.autoRenewSecs = autoRenewSecs;
 		this.memo = Optional.ofNullable(memo).orElse(DEFAULT_MEMO);
+		this.nftsOwned = nftsOwned;
```

Comment:
The property should not be part of the constructor. By default it should be simply initialised as `0`

Reply:
Yes, this should be breaking E2Es already